package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "miller-columns-element",
```

Comment:
Last chance to think if this needs namespacing on npm?

Reply:
Well.. we're already publishing `markdown-toolbar` and `paste-html-to-govspeak` without a namespace.. so I guess it may be a little late.

Reply:
Pinging @Nooshu

Reply:
I do wonder if this needs something to qualify it, this is a very specific solution for gov.uk and you'd want that to be reflected in name and/or description.

Reply:
I don't see how this is different from the other published repos `paste-html-to-govspeak` `accessible-autocomplete` or the `markdown-toolbar-element`. If we decide to qualify our published code we should review the existing one too.

Reply:
Sure, that's a reasonable argument.

I feel like paste-html-to-govspeak is fine as unless you dealing with govspeak it's not going to catch you off guard, accessible autocomplete would be fine to drop into another project. This one is a common design pattern but wouldn't help you if you found it on NPM, but I'm sure that's same problem with markdown-toolbar-element.